docs/reference/quantity_point.md

```diff
@@ -1,31 +1,422 @@
+
+??? example "Example: extracting `4.56` from `meters_pt(4.56)`"
+    Consider this `QuantityPoint<Meters, double>`:
+
+    ```cpp
+    auto p = meters_pt(4.56);
+    ```
+
+    You can retrieve the underlying value by writing either `p.in(meters_pt)` (passing the
+    `QuantityPointMaker`), or `p.in(Meters{})` (passing an instance of the unit).
+
+### `.data_in(unit)`
+
+This function returns a reference to the underlying stored value.  See the [unit
+slots](../discussion/idioms/unit-slots.md) discussion for valid choices for `unit`.
+
+??? example "Example: incrementing the underlying stored value"
+    ```cpp
+    auto temperature = celsius_pt(20);
+    temperature.data_in(celsius_pt)++;
+    ```
+
+    Since `temperature` is not `const`, the reference returned by `.data_in()` will be mutable, and
+    we can treat it like any other `int` lvalue.  The above would result in `celsius_pt(21)`.
+
+## Performing unit conversions
+
+We have two methods for performing unit conversions.  They have identical APIs, but their names are
+different.  `as` returns another `QuantityPoint`, but `in` exits the library and returns a raw
+number.
+
+### `.as(unit)`, `.as<T>(unit)` {#as}
+
+This function produces a new representation of the input `QuantityPoint`, converted to the new unit.
+See the [unit slots](../discussion/idioms/unit-slots.md) discussion for valid choices for `unit`.
+
+??? example "Example: converting `meters_pt(3)` to `centi(meters_pt)`"
+    Consider this `QuantityPoint<Meters, int>`:
+
+    ```cpp
+    auto point = meters_pt(3);
+    ```
+
+    Then `point.as(centi(meters_pt))` re-expresses this quantity in units of centimeters.
+    Specifically, it returns a `QuantityPoint<Centi<Meters>, int>`, which is equal to
+    `centi(meters_pt)(300)`.
+
+    The above example used the quantity maker, `centi(meters_pt)`.  One could also use an instance
+    of the unit type `Centi<Meters>`, writing `point.as(Centi<Meters>{})`.  The former is generally
+    preferable; the latter is mainly useful in generic code where the unit type may be all you have.
+
+**Without** a template argument, `.as(unit)` obeys the same safety checks as for the [implicit
+constructors](#implicit-from-quantity): conversions at high risk for integer overflow or truncation
+are forbidden.  Additionally, the `Rep` of the output is identical to the `Rep` of the input.
+
+**With** a template argument, `.as<T>(unit)` has two differences.
+
+1. The output `Rep` will be `T`.
+2. The conversion is considered "forcing", and will be permitted in spite of any overflow or
+   truncation risk.  The semantics are similar to `static_cast<T>`.
+
+??? example "Example: forcing a conversion from centimeters to meters"
+    `centi(meters_pt)(200).as(meters_pt)` is not allowed.  This conversion will divide the
+    underlying value, `200`, by `100`.  While this particular value would produce an integer result,
```

Comment:
Can we use an example that wouldn't divide into an integer result evenly -- perhaps 250 cm as m? Using example values that would produce a valid `int` result doesn't emphasize your point as strongly.

Reply:
I actually wanted a result that divides evenly!  I expect human readers to notice that these specific numbers would work, but I wanted to remind them that the compiler can't know the actual number when it makes the permit/forbid decision, as the program is being built.  Note that the third paragraph drives the point home by showing what would happen if we forced the conversion with an inexact multiple.

That said, I think the wording here could be much better, so I cleaned it up.  I made my emphasis more explicit, and I combined the second and third paragraphs because they're both about forcing.